docs/user-interface/playground_chat.md

```diff
@@ -0,0 +1,31 @@
+---
+title: InstructLab UI Playground
+description: Steps to set up the playground to chat with a model
+logo: images/ilab_dog.png
+---
+
+To run with a locally run model, make sure that iLab model serve is running in a seperate terminal. If you are unsure on how to do this, please visit the [Intro to serve and chat](/getting-started/serve_and_chat/) section of this document.
+
+If you go to `Playground > Chat` by using the side navigation bar, you can interact with the merlinite and granite models. 
+
+![UI No Model Response](../images/user-interface/ui_no_model_response.png)
+
+However, the model won't reply because a granite/ merlinite model endpoint hasn't been given. Instead, we will create a new custom model endpoint, using our locally hosted quantised model from step 1.
```

Comment:
@dominikkawka Focus of this document is dev environment or the direct users of ui.instructlab.ai? Details looks good for dev setup, but using the ui.instructlab.ai they can chat with both merlinite and granite models.